dpgen2/fp/vasp.py

```diff
@@ -72,7 +97,42 @@ def prep_task(
         """
 
         conf_frame.to("vasp/poscar", vasp_conf_name)
-        Path(vasp_input_name).write_text(vasp_inputs.incar_template)
+        incar = vasp_inputs.incar_template
+
+        use_ele_temp = 0
```

Comment:
It would be much better to write a function for revising the INCAR template and provide a unit test for testing if it works correctly.